cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -40,12 +40,11 @@
     %}
 {% endif %}
 {% set value = value or global_dict.expandable %}
-
 <div class="m-expandable
-            m-expandable__expanded
```

Comment:
@anselmbradford, I've removed that code and have opted to used an animation. This will only work on IE10+ however. Not sure if this is the best approach.